wesnoth.po

```diff
@@ -4330,6 +4314,8 @@ msgid ""
 "All adjacent friendly units will strike first in melee combat, even when "
 "defending."
 msgstr ""
+"Všechny sousedící spřátelené jednotky zaútočí při boji zblízka jako první, a "
+"to tehdy, když se brání."
```

Comment:
Pro mne má to **i** navíc smysl - tedy pokud jednotky útočí první i v případě, když bojují proti těm, co útočí první.
```suggestion
"to i tehdy, když se brání."
```